.github/workflows/ci.yml

```diff
@@ -20,7 +20,7 @@ jobs:
     strategy:
       matrix:
         otp: [24, 23, 22]
-        elixir: ['1.11', '1.10']
+        elixir: ['1.13', '1.12', '1.11', '1.10']
```

Comment:
Thanks for updating the versions.

I have not written down, but usually I try to aim only for the last 3 releases.
Usually sufficient and reduces the size of the build matrix.

```suggestion
        elixir: ['1.13', '1.12', '1.11']
```